src/ocm/components/clusterConfiguration/networkConfiguration/AvailableSubnetsControl.tsx

```diff
@@ -83,9 +83,9 @@ export const AvailableSubnetsControl = ({
                       name={`machineNetworks.${index}.cidr`}
                       options={
                         (index === 1 ? IPv6Subnets.length > 0 : IPv4Subnets.length > 0)
-                          ? [makeNoSubnetSelectedOption(hostSubnets)].concat(
-                              toFormSelectOptions(index === 1 ? IPv6Subnets : IPv4Subnets),
-                            )
+                          ? [
+                              makeNoSubnetSelectedOption(index === 1 ? IPv6Subnets : IPv4Subnets),
+                            ].concat(toFormSelectOptions(index === 1 ? IPv6Subnets : IPv4Subnets))
                           : [makeNoSubnetAvailableOption()]
                       }
                       isDisabled={!hostSubnets.length}
```

Comment:
@jgyselov shouldn't it be disabled iff:
```
(index === 1 ? IPv6Subnets : IPv4Subnets).length === 0
```

Reply:
FYI I refactored this code a bit in my branch for read-only as I could see a bit of code repetition and tried to remove as much as I could. We can discuss it also in that MR.
https://github.com/openshift-assisted/assisted-ui-lib/pull/1429/files#diff-29ad431060c005c7bcdd8266019c322ffcb1c603a62f607f01a406413cf5c3bcR1

Reply:
Do I see correctly, that the read-only feature is planned for v2.7?

It will a while until we get to that version, so maybe we could use this quick fix until then? @celdrake

Reply:
Yes, your change should go first as it's now. I'll adjust my PR from your changes here.

Reply:
Looking at your PR, it doesn't seem you'll need to change anything because of my fix here :slightly_smiling_face: @celdrake